src/Sarif.Driver.UnitTests/SarifHelpers.cs

```diff
@@ -32,7 +30,6 @@ public static void ValidateRun(Run run, Action<Result> resultAction, Action<Noti
 
         public static void ValidateTool(Tool tool)
         {
-            Assert.Equal("Sarif", tool.Name);
```

Comment:
great change!  Apparently this was passing before .NET core because the some code that looks at the assembly to populate tool.Name.  When running the test in .NET Core, the assembly that is returned is the test framework, hence the value of "testHost".